doc/tutorial_ti_cc3200.md

```diff
@@ -317,16 +319,20 @@ Reaching this point means you are able to produce and execute CC3200 compatible
             xi_shutdown();
         }
 
-- you can get a Account ID and Device ID from the Devices Page of CPM. To get the Password, click the Get Password button in the top right of that page.
+- you can get a Device ID and Device Secret from the Devices Page of CPM. Navigate to a specific device page and click the Get Password button in the top right of that page to download a file with the values.
+
+- locate the successful wifi connection point in the `main.c` of the ent_wlan example (around line 647, comment: "//wait for few moments"). Here put a call on the ConnectToXively(); function we just added. 
 
 - to make aboves buildable you'll need to
-    - add two include paths to your project to help compiler find xively.h and friends: ```Project```->```Properties```->```Build```->```ARM Compiler```->```Include Options```:
-        - xively-client-c/include
-        - xively-client-c/include/bsp
+    - add two include paths to your project to help the compiler find `xively.h` and friends: ```Project```->```Properties```->```Build```->```ARM Compiler```->```Include Options```:
+        - `xively-client-c/include`
```

Comment:
If you do three and then a newline it will do the code block. Just one will give you the inline code style.